qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,510 @@
+                                        "off": 0,
+                                        "on": 1,
+                                        "auto": 2},
+                           get_cmd=self._get_output,
+                           set_cmd=self._set_output)
+
+        # Set actual unit (either mm or m°) to positional parameters
+        self._update_position_unit()
+
+    # Version 3
+    # ---------
+    def single_step(self, backward: Optional[Union[bool, str, int]] = None) -> None:
+        """
+        Triggers a single step in desired direction.
+
+        Args:
+            backward: Step direction forward (False) or backward (True). Beside True/False, you can
+                      set the direction to "forward"/"backward" or +1/-1 (default: forward or False)
+        """
+        backward = self._map_direction_parameter(backward)
+
+        self._parent._lib.start_single_step(self._parent._device_handle, self._axis, backward)
+
+    def multiple_steps(self, steps: int) -> None:
+        """
+        Performs multiple steps. The direction depends on the sign (+: forward, -: backward)
+
+        Args:
+            steps: Number of steps to move. The sign indicates the moving direction (+: forward,
+                   -: backward)
+        """
+        backward = (steps < 0)
+        
+        for i in range(abs(steps)):
+            self.single_step(backward)
+
+    def start_continuous_move(self, backward: Optional[Union[bool, str, int]] = None):
+        """
+        Starts continuous motion in forward or backward direction.
+        Other kinds of motion are stopped.
+
+        Args:
+            backward: Step direction forward (False) or backward (True). Beside True/False, you can
+                      set the direction to "forward"/"backward" or +1/-1 (default: forward or False)
+        """
+        backward = self._map_direction_parameter(backward)
+
+        self._parent._lib.start_continuous_move(self._parent._device_handle, self._axis, True,
+                                                backward)
+
+    def stop_continuous_move(self):
+        """Stops continuous motion in forward or backward direction."""
+        self._parent._lib.start_continuous_move(self._parent._device_handle, self._axis, False,
+                                                False)
+
+    _direction_mapping: Dict[Any, bool] = {
+        "forward": False,
+        "backward": True,
+        +1: False,
+        -1: True
+    }
+
+    @classmethod
+    def _map_direction_parameter(cls, backward: Optional[Union[bool, str, int]]) -> bool:
+        if backward is None:
+            return False
+        if backward in [False, True]:
+            return bool(backward)
+
+        if backward in cls._direction_mapping:
+            return cls._direction_mapping[backward]
+
+        raise ValueError("Unexpected value for argument `backward`. Allowed values are: {}".format(
+            [None, False, True, *cls._direction_mapping.keys()]))
```

Comment:
This format is preferred:
raise ValueError(f"Unexpected value for argument `backward`. Allowed values are:"
                         f"{[None, False, True, *_direction_mapping.keys()]}")